Source/ID3TagEditor.swift

```diff
@@ -38,7 +39,7 @@ public class ID3TagEditor {
      Could throw `CorruptedFile` if the file is corrupted.
      */
     public func read(from path: String) throws -> ID3Tag? {
-        let mp3 = try mp3FileReader.readFrom(path: path)
+        let mp3 = try mp3FileReader.readID3HeaderFrom(path: path)
```

Comment:
Can we rename the variable mp3 to `currentTag`. We can also probably rename the parameter of the `id3TagParser`. What do you think?

Reply:
I am unsure about this. Naming it `currentTag` would be a bit confusing. 

If I take a look at the `write` method:

```swift
    public func write(tag: ID3Tag, to path: String, andSaveTo newPath: String? = nil) throws {
        let mp3 = try mp3FileReader.readFileFrom(path: path)
        let currentTag = try self.id3TagParser.parse(mp3: mp3)
        let mp3WithId3Tag = try mp3WithID3TagBuilder.build(mp3: mp3, newId3Tag: tag, currentId3Tag: currentTag)
        try mp3FileWriter.write(mp3: mp3WithId3Tag, path: newPath ?? path)
    }
```

Here, `currentTag` is used to hold an `ID3Tag` type. If we used the same `currentTag` name here, to hold a `Data` type, it would be pretty confusing. 

What do you think about `tagData`? 🤔 

----

I would prefer to keep the name `mp3` in **this** PR, and I can make another PR to change it.

Reason being, if we rename the parameter in `id3TagParser`, so it would read `func parse(tagData: Data) throws -> ID3Tag?`, it would probably make sense to rename it everywhere, no?
- ID3FrameSizeParser: `func parse(mp3: NSData, framePosition: Int, version: ID3Version) -> Int`
- ID3FramesParser: `func parse(mp3: NSData, id3Tag: ID3Tag)`

This would make this PR pretty huge. 

And there's also the public facing function in `ID3TagEditor.swift`:
`public func read(mp3: Data) throws -> ID3Tag?`

It would probably make sense here as well to use the same term, to refer to the `Data` type.

But this would be a breaking change. 🤔

Reply:
Ok now problem (also because this can be improved with some tagged types created ad hoc in the future)